flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/runtime/util/collections/ByteHashSet.java

```diff
@@ -17,21 +17,17 @@
 
 package org.apache.flink.table.runtime.util.collections;
 
-import org.apache.flink.table.util.MurmurHashUtil;
-
 /**
  * Byte hash set.
  */
 public class ByteHashSet extends OptimizableHashSet {
 
-	private byte[] key;
-
-	private byte min = Byte.MAX_VALUE;
-	private byte max = Byte.MIN_VALUE;
+	private final byte min = Byte.MIN_VALUE;
+	private final byte max = Byte.MAX_VALUE;
 
 	public ByteHashSet(final int expected, final float f) {
 		super(expected, f);
-		this.key = new byte[this.n + 1];
+		used = new boolean[max - min + 1];
 	}
 
 	public ByteHashSet(final int expected) {
```

Comment:
Just leave behind this `public ByteHashSet(final int expected)`, remove other useless constructor.

Reply:
I have removed all but the default constructor, because it seems the 'expected' parameter is not used in the code.